appinfo/info.xml

```diff
@@ -40,4 +40,9 @@
 			<provider>OCA\Approval\Activity\ApprovalProvider</provider>
 		</providers>
 	</activity>
+	<repair-steps>
+		<post-migration>
```

Comment:
My only comment is: This should really only ever be called once for version 1.0.13. May be easier to guarantee with an actual db migration (without changing the schema and just using the post schema change method)

Reply:
Yeah that's a better approach

Reply:
@nickvergessen @marcelklehr Done.